src/micrate.cr

```diff
@@ -140,7 +143,7 @@ module Micrate
   private def self.migrations_by_version(migrations_path)
     Dir.entries(migrations_path)
        .select { |name| File.file? File.join(migrations_path, name) }
-       .select { |name| /^\d+_.+\.sql$/ =~ name }
+       .select { |name| /^\d+.+\.sql$/ =~ name }
```

Comment:
The mistake I made was to use a dash instead of an underscore. I realise you want a timestamp, so the options are....

`/^\d+_.+\.sql$/` as now requiring an underscore
`/^\d+[_-].+\.sql$/` requiring dash or underscore
`/^\d{,14}.+\.sql$/` requiring 14 numbers ie yyyymmddhhmmss
`/^\d+.+\.sql$/` anything as long as it starts with a number and ends with .sql

I agree that there should be a format, but like to avoid punishing users for difficult to spot mistakes. The 14 number version is possibly the best solution as this is based on what you need from them. Drop me an update and I can change / remove this...

Thanks.